### DISCUSSION TITLE
How to monitor multiple RPi's

### DISCUSSION THREAD

**Discussion by LabeebQa** [carlosedp/cluster-monitoring#122] [General]
Hi,

I would like to monitor multiple RPi's that arent in a cluster. Please explain how to do this.I am pretty much a newbie to this

**Comment by carlosedp** [comment]
This stack is meant to monitor a Kubernetes cluster. If you need to monitor multiple external hosts, I'd recommend deploying a Prometheus and Grafana and to each host node-exporter.
